server/controllers/booking.js

```diff
@@ -0,0 +1,71 @@
+const Booking = require('../models/Booking');
+const asyncHandler = require('express-async-handler');
+
+// @route GET /booking//bookings
+// @desc get all bookings of the sitter who is logged in
+exports.getAllBookings = asyncHandler(async (req, res, next) => {
+  const bookings = await Booking.find({ sitterId: req.params.id });
+  res.send(bookings);
+});
+
+// @route POST /booking//book
```

Comment:
this would require that controller moved to another file so it would have a different path.
const bookingRouter = require('./routes/booking');
`app.use('/booking', bookingRouter);`
is that what you want me to do?